Tools/scripts/uploader.py

```diff
@@ -840,6 +898,10 @@ def main():
                     # Go to the next port
                     continue
 
+                if args.identify:
+                    up.dump_board_info()
+                    sys.exit(0)
```

Comment:
Does this close the port (meaning does finally part run after the exit)?

In any case, I think this should be inside the try below and just be an additional option (without the exit part):

```
if args.identify:
  up.dump_board_info()
else if args.download:
 [...]
else:
 [...]
```

Reply:
Done.